### PR TITLE
Added conversations page

### PR DESCRIPTION
**Pull request by NikoVe96**

### CODE REVIEW

**Comment by NikoVe96** [comment]
@mircealungu

File: src/Conversations.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { Link } from 'react-router-dom';
+
+function Conversations() {
+    return (
+        <>
+            <div className="container">
+                <h1>Conversations</h1>
+                <div className="centered-content-Conversations">
+                    <figure>
+                        <image src="Images/man-avatar.png" alt="avatar" width="60" height="60" class="img"></image>
```
Comment (igawaclawska) [inline src/Conversations.js:12]: The `<image>` tag is not a valid tag. Use `` instead.
Replace `class` with `className`.

File: src/MainPage.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    <image className="logoImage" src="/src/images/logo.jpg" alt="Logo"></image>
+                    <h1> Elder Friendly Chat</h1>
+
+                    <div className="centered-buttons">
+                        <img className="img" src="/src/images/conversations1.png" alt="conversations"></img>
+                        <Link to="/conversations"><button class="button"> View conversations</button></Link>
+                    </div>
+
+                    <div className="centered-buttons">
+                        <img className="img" src="/src/images/add_contact.png" alt="conversations"></img>
```
Comment (igawaclawska) [inline src/MainPage.js:19]: I've observed that there seems to be a problem with displaying images in the app (At least on my end). 

To fix it, try the following solution:

- Go to the "public" directory and create an " images " folder in there.

- Place all visual assets in this newly created "images" folder.

- Whenever you need to use images in your components or pages, use this pattern for their paths:  `src="images/image_name.format"`
(For instance:`conversations</img>`)

Let me know if it helped :)

File: src/MainPage.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                        <button className="button">Add new contact</button>
+                    </div>
+                </div>
+
+                <div>
+                    <button className="settingsButton"> Settings</button>
```
Comment (igawaclawska) [inline src/MainPage.js:25]: Consider using kebab-case for CSS selectors (e.g., settings-button)
https://github.com/itu-tid/lecture-notes/blob/main/Tech-TopUps/5-Coding-Practices.md#name-your-css-classes-with-kebab-case

**Review by igawaclawska — CHANGES_REQUESTED** [review]
Looks good!
I've added a few change requests. Try to implement them and tag us once you are done :)

**Pull request closed without merging**